src/views/profile/changeavatar.ejs

```diff
@@ -56,12 +56,12 @@
             <li>Must not be inappropriate.</li>
             <li>Must have similar Res and Spy avatars.</li>
             <li>Must not be similar to any other existing custom avatars (unless the avatar was yours initially).</li>
-            <li>Must be 128x128 px in size.</li>
+            <li>Must be <%= VALID_DIMENSIONS_STR %> in size.</li>
+            <li>Must be maximum <%= MAX_FILESIZE_STR %> in size.</li>
             <li>Must be a png file.</li>
             <li>Must have a transparent background (solid white will hurt the eyes of dark theme users!).</li>
             <li>Must resemble the base avatar images.</li>
             <li>Must keep the wordage to a minimum.</li>
-            <li>Must be hosted on Discord or Imgbb. Click <a href="/profile/avatargetlinktutorial">here</a> for a tutorial on how to get a good link.</li>
             <li>Moderators have the final say, based on their judgment. Feel free to consult them before making your avatars to ensure they will adhere to these guidelines.</li>
```

Comment:
If you can, let's pull these duplicate pieces out to a separate file under `partials` or somewhere you like, and then include it in directly.